charter.md

```diff
@@ -53,15 +53,164 @@ The following responsibilities belong to the Steering Committe, but may be deleg
 * Control and delegate access to and establish processes regarding
   project resources/assets<sup>[6](#footnote6)</sup>  
 
-## Quorum
+## Changes
 
-Steering committee members are generally expected to attend every meeting. We
+In instances where a process is not already specified within this document,
+changes to the Steering Committee charter will be considered according to the
+processes set forth in the committee's [operations documentation][changes].
+
+[changes]: /operations/changes.md
+
+## Membership
+
+### Elections
+
+<!--
+
+- Consider explicitly mentioning SIG ContribEx's Elections subproject, once it
+exists.
+
+-->
+
+Every year, the Steering Committee holds a general election for open seats with
+[SIG Contributor Experience][sig-contribex] owning the process.
+
+Our [election policy document][general-elections] covers the details for how
+this works.
+
+In the instance that a general election is called as a result of the
+[dissolution](#dissolution) of the current committee, the following rules will
+apply:
+
+* members of the current committee will not be allowed to run in the new
+  election
+* a new committee will be bootstrapped with:
+  * the top four candidates serving a two-year term
+  * the next three candidates serving a one-year team
+
+Policy and operations of special elections initiated by the Steering Committee
+for other reasons (as yet to be determined) may be handled by some agreed-upon
+[process][changes] by the committee.
+
+[general-elections]: /elections.md
+[sig-contribex]: https://git.k8s.io/community/sig-contributor-experience
+
+### Vacancies
+
+In the event of a resignation, removal, or other loss of an elected Steering 
+Committee member, the candidate with the next most votes from the previous 
+election will be offered the seat. This process will continue until the seat is 
+filled. 
+
+In case this process fails to fill the seat, a special election for that position will
+be held as soon as possible. [Eligible voters](https://git.k8s.io/steering/elections.md#eligibility-for-voting) from the most recent election 
+will vote in the special election (ie: eligibility will not be redetermined at 
+the time of the special election). A committee member elected in a special 
+election will serve out the remainder of the term for the person they are 
+replacing, regardless of the length of that remainder.
+
+### Resignation
+
+If a committee member chooses not to continue in their role, for whatever
+self-elected reason, they must notify the committee in writing.
+
+### Dissolution
```

Comment:
The point of the staggered terms is so the board has continuity; both of these options would lose that. I have been on a board in the past and continuity is super super important.

I would consider adding something like:

#### Continuity

If the board is dissolved completely by any mechanism, selected members (selection described below) agree to be "continuity advisors" and attend meetings of the subsequently elected incoming board in a non-voting capacity for 6 meetings or until the incoming board dismisses them, whichever is first.

Selection. At least two and not more than three members shall be selected for this duty.
* One shall be the longest-serving outgoing board member. If multiple, the incoming board may select between them.
* The second shall be selected by the incoming board at their discretion.
* The incoming board, at their discretion, may select a third.

Reply:
+1

Reply:
> The point of the staggered terms is so the board has continuity; both of these options would lose that. I have been on a board in the past and continuity is super super important.

Great point. Added this language in https://github.com/kubernetes/steering/pull/248/commits/e0ffe089051dd731f33b19f1e32a19a8a25a0094.